### PR TITLE
Add supporting default alert button styles

### PR DESCRIPTION
**Pull request by dukhovnyi**
Introducing applying alert button styles as a default for survey buttons.

### CODE REVIEW

**Review by kuldar-daniel — APPROVED** [review]

File: GliaWidgets/ViewController/Survey/SurveyViewController.View.swift [hunk 1]
```diff
@@ -162,3 +150,25 @@ extension NSAttributedString {
         return mutableString
     }
 }
+
+extension UIButton {
+    func update(with style: Theme.Button) {
+        style.background.map { backgroundColor = UIColor(hex: $0) }
```
Comment (igorkravchenko) [inline GliaWidgets/ViewController/Survey/SurveyViewController.View.swift:156]: While it's convenient to use map like that, I think it goes against the purpose of map operation (transforming values).
Is it possible to use it like this?
```
backgroundColor = style.background.map(UIColor.init(hex:))
```
Does it make sense?

File: GliaWidgets/ViewController/Survey/SurveyViewController.View.swift [hunk 2]
```diff
@@ -162,3 +150,25 @@ extension NSAttributedString {
...
+        contentEdgeInsets = .init(top: 6, left: 16, bottom: 6, right: 16)
+        clipsToBounds = true
+        layer.masksToBounds = false
+        layer.cornerRadius = style.cornerRadius
+
+        style.borderColor.map { layer.borderColor = UIColor(hex: $0).cgColor }
```
Comment (igorkravchenko) [inline GliaWidgets/ViewController/Survey/SurveyViewController.View.swift:162]: Same comment as above.

**Review by igorkravchenko — COMMENTED** [review]
Looks good. Just a few map related questions.

File: GliaWidgets/ViewController/Survey/SurveyViewController.View.swift (hunk 1 above)
Comment (dukhovnyi) [reply GliaWidgets/ViewController/Survey/SurveyViewController.View.swift:156]: image
I thought the same, but for some reason, Xcode throws error...

**Review by igorkravchenko — APPROVED** [review]

**Pull request closed without merging**